### PR TITLE
Daniel wu/robocup sw tutorial

### PR DESCRIPTION
**Pull request by dwu359**
Screen Shot 2022-11-06 at 8 24 53 PM
Screen Shot 2022-11-06 at 8 25 24 PM

Stuff that confused me in this tutorial: Could be more clear on what Roles, Tactics, and Plays are in step 3, could also be more clear on what launch files/parts of the cmake files that should be modified for step 6

### CODE REVIEW

File: .vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            "defines": [],
+            "compilerPath": "/usr/bin/clang",
+            "cStandard": "c17",
+            "cppStandard": "c++14",
+            "intelliSenseMode": "linux-clang-arm64",
+            "configurationProvider": "ms-vscode.cmake-tools"
+        }
+    ],
+    "version": 4
+}
```
Comment (p-nayak11) [inline .vscode/c_cpp_properties.json:21]: Make sure that the changes that you make for your vscode configs are not pushed to the main branch.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "files.associations": {
+        "*.idl": "cpp"
+    },
+    "ros.distro": "foxy"
+}
```
Comment (p-nayak11) [inline .vscode/settings.json:6]: Same as above.

File: rj_gameplay/rj_gameplay/role/runner_role.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import numpy as np
+
+class RunnerRole(stp.role.Role):
+    """"""
+
+    def __init__(self, robot: stp.rc.Robot) -> None:
+        self.curr = 0
+        super().__init__(robot)
+
+    def tick(self, world_state: stp.rc.WorldState) -> RobotIntent:
```
Comment (p-nayak11) [inline rj_gameplay/rj_gameplay/role/runner_role.py:14]: When creating new projects in the future, please adhere to the doc strings for the language. You can find these in our Contributions page.

File: soccer/src/soccer/soccer_mom.cpp [hunk 4]
```diff
@@ -0,0 +1,20 @@
+#include <rclcpp/rclcpp.hpp>
+#include "soccer_mom.hpp"
+
+
```
Comment (p-nayak11) [inline soccer/src/soccer/soccer_mom.cpp:4]: Same comment as above. It is extremely important on the C++ aspect of the code base as we don't have too much documentation on it the first place.

**Review by p-nayak11 — CHANGES_REQUESTED** [review]
It looks good so far! Good job on completing steps 3, 5, and 6. Please add step 4 in this PR as well.

**Comment by dwu359** [comment]
> It looks good so far! Good job on completing steps 3, 5, and 6. Please add step 4 in this PR as well.

For step 4, the file is /rj_msgs/msg/RobotIntent.msg

**Review by p-nayak11 — APPROVED** [review]
LGTM

**Pull request closed without merging**